Product.jsx

```diff
@@ -0,0 +1,14 @@
+require("./node_modules/materialize-css/dist/css/materialize.min.css")
+import React from 'react';
+
+export default class Product extends React.Component {
```

Comment:
At this point, the product should display a name and a release date. Also, why not render them as `<li>`?